### ISSUE TITLE
Issue while installing Hbase

### ISSUE THREAD

**Issue by PRIYANKAPANDEYTWO** [naver/pinpoint#2460]
Hi Team,

Need Your help actually want to start exploring Pinpoint while installing Pinpoint get failed multiple place like jboss,collector but able to resolve as those were  SSL issue went in pom.xml and convert https:// into http:// for jboss and cloudera. So we done with installation of Pinpoint with Maven.Next step is "Download & Start - Run quickstart/bin/start-hbase.sh" but we get failed with hbase start 

#### Description
As per instruction For Windows(Windows 7), you'll have to download HBase manually from Apache download site.
Download HBase-1.0.3-bin.tar.gz and unzip it. We did same and next step was to Run quickstart/bin/start-hbase.Download completed but get failed with start step

#### Environment

* Pinpoint version:1.6.1
* Windows: 7

#### Additional Info
Attaching Logs quickstart.collector

[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.navercorp.pinpoint:pinpoint-quickstart-collector:war:1.6.1-SNAPSHOT
[WARNING] 'version' contains an expression but should be a constant. @ com.navercorp.pinpoint:pinpoint-quickstart-collector:${maven.pinpoint.version}, C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\pom.xml, line 11, column 14
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-war-plugin is missing. @ com.navercorp.pinpoint:pinpoint-quickstart-collector:${maven.pinpoint.version}, C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\pom.xml, line 41, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Pinpoint Collector QuickStart 1.6.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ pinpoint-quickstart-collector ---
[INFO] Deleting C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\target
[INFO] 
[INFO] --- properties-maven-plugin:1.0-alpha-2:read-project-properties (default) @ pinpoint-quickstart-collector ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ pinpoint-quickstart-collector ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\src\main\java
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ pinpoint-quickstart-collector ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ pinpoint-quickstart-collector ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\src\test\resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ pinpoint-quickstart-collector ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ pinpoint-quickstart-collector ---
[INFO] No tests to run.
[INFO] 
[INFO] --- maven-war-plugin:2.2:war (default-war) @ pinpoint-quickstart-collector ---
[INFO] Packaging webapp
[INFO] Assembling webapp [pinpoint-quickstart-collector] in [C:\Users\PXP8422\pinpoint\quickstart\collector\target\deploy]
[INFO] Processing war project
[INFO] Processing overlay [ id com.navercorp.pinpoint:pinpoint-collector]
[INFO] Webapp assembled in [3938 msecs]
[INFO] Building war: C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\target\pinpoint-quickstart-collector-1.6.1-SNAPSHOT.war
[INFO] WEB-INF\web.xml already added, skipping
[INFO] 
[INFO] >>> tomcat7-maven-plugin:2.2:run (default-cli) > process-classes @ pinpoint-quickstart-collector >>>
[INFO] 
[INFO] --- properties-maven-plugin:1.0-alpha-2:read-project-properties (default) @ pinpoint-quickstart-collector ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ pinpoint-quickstart-collector ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\src\main\java
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ pinpoint-quickstart-collector ---
[INFO] No sources to compile
[INFO] 
[INFO] <<< tomcat7-maven-plugin:2.2:run (default-cli) < process-classes @ pinpoint-quickstart-collector <<<
[INFO] 
[INFO] --- tomcat7-maven-plugin:2.2:run (default-cli) @ pinpoint-quickstart-collector ---
[INFO] Running war on http://localhost:28082/
[INFO] Creating Tomcat server configuration at C:\Users\PXP8422\pinpoint\quickstart\bin\..\collector\target\tomcat
[INFO] create webapp with contextPath: 
2017-01-09 16:40:46 [INFO ](o.s.w.c.ContextLoader              :285) Root WebApplicationContext: initialization started
2017-01-09 16:40:46 [INFO ](o.s.w.c.s.XmlWebApplicationContext :514) Refreshing Root WebApplicationContext: startup date [Mon Jan 09 16:40:46 IST 2017]; root of context hierarchy
2017-01-09 16:40:46 [INFO ](o.s.b.f.x.XmlBeanDefinitionReader  :317) Loading XML bean definitions from class path resource [applicationContext-collector.xml]
2017-01-09 16:40:47 [INFO ](o.s.b.f.x.XmlBeanDefinitionReader  :317) Loading XML bean definitions from class path resource [applicationContext-hbase.xml]
2017-01-09 16:40:47 [INFO ](b.f.c.PropertyPlaceholderConfigurer:172) Loading properties file from class path resource [hbase.properties]
2017-01-09 16:40:47 [INFO ](b.f.c.PropertyPlaceholderConfigurer:172) Loading properties file from class path resource [pinpoint-collector.properties]
2017-01-09 16:40:47 [INFO ](utowiredAnnotationBeanPostProcessor:153) JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
2017-01-09 16:40:47 [INFO ](c.n.p.c.h.PooledHTableFactory      :71 ) create HConnectionThreadPoolExecutor poolSize:128, workerQueueMaxSize:5120
2017-01-09 16:40:47 [WARN ](o.a.h.u.NativeCodeLoader           :62 ) Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2017-01-09 16:40:47 [ERROR](o.a.h.u.Shell                      :373) Failed to locate the winutils binary in the hadoop binary path
java.io.IOException: Could not locate executable null\bin\winutils.exe in the Hadoop binaries.
	at org.apache.hadoop.util.Shell.getQualifiedBinPath(Shell.java:355)
	at org.apache.hadoop.util.Shell.getWinUtilsPath(Shell.java:370)
	at org.apache.hadoop.util.Shell.<clinit>(Shell.java:363)
	at org.apache.hadoop.util.StringUtils.<clinit>(StringUtils.java:78)
	at org.apache.hadoop.security.Groups.parseStaticMapping(Groups.java:93)
	at org.apache.hadoop.security.Groups.<init>(Groups.java:77)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:240)
	at org.apache.hadoop.security.Groups.getUserToGroupsMappingService(Groups.java:225)
	at org.apache.hadoop.hbase.security.UserProvider.<clinit>(UserProvider.java:56)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:214)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:150)
	at com.navercorp.pinpoint.common.hbase.PooledHTableFactory.<init>(PooledHTableFactory.java:59)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:147)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:122)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:267)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1137)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1040)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:504)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1220)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1120)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1044)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:942)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:533)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1208)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:762)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:434)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:306)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:106)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4939)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5434)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1559)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1549)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2017-01-09 16:40:48 [INFO ](o.a.h.h.z.RecoverableZooKeeper     :120) Process identifier=hconnection-0x6a6c15 connecting to ZooKeeper ensemble=localhost:2181
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:host.name=ATCVMDEVU17014.amer.homedepot.com
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.version=1.8.0_45
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.vendor=Oracle Corporation
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.home=C:\Program Files\Java\jdk1.8.0_45\jre
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.class.path=C:\Program Files\apache\maven\boot\plexus-classworlds-2.5.2.jar
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.library.path=C:\Program Files\Java\jdk1.8.0_45\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files\Java\jdk1.8.0_45\bin;C:\Program Files\Java\jdk1.7.0_80\bin;C:\Program Files\Java\jdk1.6.0_45\bin;C:\Program Files\apache\maven\bin;c:\opt\isv\devtools\Perl\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\Common Files\ThinPrint\;.
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.io.tmpdir=C:\Users\PXP8422\AppData\Local\Temp\
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:java.compiler=<NA>
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:os.name=Windows 7
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:os.arch=x86
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:os.version=6.1
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:user.name=PXP8422
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:user.home=C:\Users\PXP8422
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :100) Client environment:user.dir=C:\WINDOWS\system32
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ZooKeeper          :438) Initiating client connection, connectString=localhost:2181 sessionTimeout=90000 watcher=hconnection-0x6a6c150x0, quorum=localhost:2181, baseZNode=/hbase
2017-01-09 16:40:48 [INFO ](o.a.h.h.s.o.a.z.ClientCnxn         :975) Opening socket connection to server 127.0.0.1/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
2017-01-09 16:40:49 [WARN ](o.a.h.h.s.o.a.z.ClientCnxn         :1102) Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)

**Comment by Xylus** [comment]
Hi @PRIYANKAPANDEYTWO 
It looks like your hbase hasn't started up property. Please make sure your hbase is up and running properly before starting the collector/web.
You also need to run `init-hbase` after running `start-hbase` to create the tables.

**Comment by PRIYANKAPANDEYTWO** [comment]
Thanks Xylus,

I already try to run init-hbase after running start-hbase to create the tables but got below exception:-

org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'spanChunkHandler': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.navercorp.pinpoint.collector.dao.TraceDao com.navercorp.pinpoint.collector.handler.SpanChunkHandler.traceDao; nested exception is org.
springframework.beans.factory.BeanCreationException: Error creating bean with name 'hbaseTraceDaoFactory': FactoryBean threw exception on object creation; nested exception is com.navercorp.pinpoint.common.hbase.HbaseSystemException: Can't get the locations; nested exception is org.apache.hadoop.hbase.client.RetriesExhaustedException

**Comment by YouriAndropov** [comment]
Why do you use a snapshot version ? You shouldn't be surprised to have issues ...

**Comment by Xylus** [comment]
@PRIYANKAPANDEYTWO 
That looks like an error from your collector log.
There are 2 tables that was added in 1.6.0 - *AgentStatV2* and *TraceV2*, and I think the collector's failing to start because these tables aren't there.
Make sure these tables are present and give it a try again.

**Comment by PRIYANKAPANDEYTWO** [comment]
Thanks Team for all your support we are able to make PinPoint Up:-
![pinpoint up](https://cloud.githubusercontent.com/assets/25005033/22093729/f09e2804-de2d-11e6-8d05-6875c4cb9c04.PNG)

**Comment by biswa94** [comment]
Please help, after installing pinpoint,i am unable to start the hbase.
i don't know how to start them. and my pinpoint installation also got build failure.... please help me out.



[INFO] Reactor Summary:
[INFO]
[INFO] pinpoint ........................................... SUCCESS [  4.399 s]
[INFO] pinpoint-annotations ............................... SUCCESS [  3.449 s]
[INFO] pinpoint-thrift .................................... SUCCESS [  7.575 s]
[INFO] pinpoint-commons ................................... SUCCESS [  5.215 s]
[INFO] pinpoint-bootstrap-core ............................ SUCCESS [  2.689 s]
[INFO] pinpoint-bootstrap ................................. SUCCESS [  0.749 s]
[INFO] pinpoint-rpc ....................................... SUCCESS [  2.244 s]
[INFO] pinpoint-profiler .................................. SUCCESS [  6.737 s]
[INFO] pinpoint-commons-hbase ............................. SUCCESS [  2.318 s]
[INFO] pinpoint-commons-server ............................ SUCCESS [  6.758 s]
[INFO] pinpoint-profiler-test ............................. SUCCESS [  1.465 s]
[INFO] pinpoint-profiler-optional-parent .................. SUCCESS [  1.027 s]
[INFO] pinpoint-profiler-optional-jdk6 .................... SUCCESS [  0.739 s]
[INFO] pinpoint-profiler-optional-jdk7 .................... SUCCESS [  0.630 s]
[INFO] pinpoint-profiler-optional ......................... SUCCESS [  0.317 s]
[INFO] pinpoint-httpclient3-plugin ........................ SUCCESS [  1.205 s]
[INFO] pinpoint-httpclient4-plugin ........................ SUCCESS [  0.876 s]
[INFO] pinpoint-ning-asynchttpclient-plugin ............... SUCCESS [  0.738 s]
[INFO] pinpoint-gson-plugin ............................... SUCCESS [  0.439 s]
[INFO] pinpoint-cubrid-jdbc-driver-plugin ................. SUCCESS [  0.555 s]
[INFO] pinpoint-mysql-jdbc-driver-plugin .................. SUCCESS [  0.480 s]
[INFO] pinpoint-mariadb-jdbc-driver-plugin ................ SUCCESS [  0.430 s]
[INFO] pinpoint-postgresql-jdbc-driver-plugin ............. SUCCESS [  0.583 s]
[INFO] pinpoint-jtds-plugin ............................... SUCCESS [  0.461 s]
[INFO] pinpoint-oracle-jdbc-driver-plugin ................. SUCCESS [  0.770 s]
[INFO] pinpoint-commons-dbcp-plugin ....................... SUCCESS [  0.474 s]
[INFO] pinpoint-commons-dbcp2-plugin ...................... SUCCESS [  0.585 s]
[INFO] pinpoint-hikaricp-plugin ........................... SUCCESS [  0.451 s]
[INFO] pinpoint-test ...................................... FAILURE [  0.935 s]
[INFO] pinpoint-cassandra-driver-plugin ................... SKIPPED
[INFO] pinpoint-jdk-http-plugin ........................... SKIPPED
[INFO] pinpoint-redis-plugin .............................. SKIPPED
[INFO] pinpoint-tomcat-plugin ............................. SKIPPED
[INFO] pinpoint-json-lib-plugin ........................... SKIPPED
[INFO] pinpoint-jackson-plugin ............................ SKIPPED
[INFO] pinpoint-thrift-plugin ............................. SKIPPED
[INFO] pinpoint-user-plugin ............................... SKIPPED
[INFO] pinpoint-arcus-plugin .............................. SKIPPED
[INFO] pinpoint-google-httpclient-plugin .................. SKIPPED
[INFO] pinpoint-jetty-plugin .............................. SKIPPED
[INFO] pinpoint-spring-plugin ............................. SKIPPED
[INFO] pinpoint-spring-boot-plugin ........................ SKIPPED
[INFO] pinpoint-ibatis-plugin ............................. SKIPPED
[INFO] pinpoint-mybatis-plugin ............................ SKIPPED
[INFO] pinpoint-okhttp-plugin ............................. SKIPPED
[INFO] pinpoint-log4j-plugin .............................. SKIPPED
[INFO] pinpoint-logback-plugin ............................ SKIPPED
[INFO] pinpoint-dubbo-plugin .............................. SKIPPED
[INFO] pinpoint-activemq-client-plugin .................... SKIPPED
[INFO] pinpoint-cxf-plugin ................................ SKIPPED
[INFO] pinpoint-hystrix-plugin ............................ SKIPPED
[INFO] pinpoint-jboss-plugin .............................. SKIPPED
[INFO] pinpoint-vertx-plugin .............................. SKIPPED
[INFO] pinpoint-resin-plugin .............................. SKIPPED
[INFO] pinpoint-plugins ................................... SKIPPED
[INFO] pinpoint-bootstrap-core-optional ................... SKIPPED
[INFO] pinpoint-tools ..................................... SKIPPED
[INFO] pinpoint-agent-distribution ........................ SKIPPED
[INFO] pinpoint-collector ................................. SKIPPED
[INFO] pinpoint-profiler-optional-jdk8 .................... SKIPPED
[INFO] pinpoint-web ....................................... SKIPPED
[INFO] pinpoint-hbase-distribution ........................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 56.368 s
[INFO] Finished at: 2017-05-17T17:57:17+05:30
[INFO] Final Memory: 42M/161M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project pinpoint-test: Compilation failure
[ERROR] error: error reading C:\Users\Biswas\.m2\repository\org\apache\maven\maven-model-builder\3.2.5\maven-model-builder-3.2.5.jar; error in opening zip file

**Comment by Xylus** [comment]
@biswa94 Please create a new issue with more detail about your problem (fill out the template). Running maven with the debug flag (-X) will also tell you more about why the maven build is failing.
We can't figure out what's wrong with what you've posted.
Thanks.
